homeassistant/components/plugwise/number.py

```diff
@@ -70,10 +70,11 @@ async def async_setup_entry(
     entities: list[PlugwiseNumberEntity] = []
     for device_id, device in coordinator.data.devices.items():
         for description in NUMBER_TYPES:
-            if description.key in device and "setpoint" in device[description.key]:
-                entities.append(
-                    PlugwiseNumberEntity(coordinator, device_id, description)
-                )
+            if device.get(description.key) is not None:
+                if device[description.key].get("setpoint") is not None:
```

Comment:
I'll fix this after work.

Reply:
Hmm, putting back the walrus results in a mypy-error.

Reply:
As recalled per https://github.com/home-assistant/core/pull/93416#discussion_r1238426391 this would bring back the `literal-required` but sifting through the components tree there are (unfortunately?) a number of likewise ignores both for operator and literal-required.

I think @bouwew put in the effort to show all three options, we either should cherry pick the best option or would need some additional assistance or guidance. Please let us know how we can continue.

Reply:
CI seems to bug out now, but it this code wise what you meant @epenet?